source/GooglePlayServicesProject.cshtml

```diff
@@ -60,14 +60,14 @@
 
   <PropertyGroup>
     <PackageId>@(Model.NuGetPackageId)</PackageId>
-    <Title>Xamarin.Android Bindings for Google Play Services -  @(name_short)</Title>
-    <Summary>Xamarin.Android Bindings for Google Play Services - @(name_short) @(Model.NuGetVersion) artifact=@(Model.MavenGroupId):@(Model.Name) artifact_versioned=@(Model.MavenGroupId):@(Model.Name):@(artifact_version)</Summary>
+    <Title>.NET for Android (formerly Xamarin.Android) Bindings for Google Play Services -  @(name_short)</Title>
+    <Summary>.NET for Android (formerly Xamarin.Android) Bindings for Google Play Services - `@(name_short)` `@(Model.NuGetVersion)`</Summary>
     <Description>
-        Xamarin.Android Bindings for Google Play Services - @(name_short) @(Model.NuGetVersion) artifact=@(Model.MavenGroupId):@(Model.Name) artifact_versioned=@(Model.MavenGroupId):@(Model.Name):@(artifact_version)
+        .NET for Android (formerly Xamarin.Android) Bindings for Google Play Services - `artifact_versioned=@(Model.MavenGroupId):@(Model.Name):@(artifact_version)`
```

Comment:
Like https://github.com/xamarin/AndroidX/pull/623/commits/9ba69b350efe85b4e0cd22ad9891c68762c0a042, let's remove the `artifact=` and `artifact_versioned=` stuff from the `<Description>` since that isn't really intended for humans.

Reply:
ok

Reply:
removed from all XML nodes, but `PackageTags`